src/openrct2/scripting/ScRide.hpp

```diff
@@ -143,10 +637,31 @@ namespace OpenRCT2::Scripting
             dukglue_register_property(ctx, &ScRide::object_get, nullptr, "object");
             dukglue_register_property(ctx, &ScRide::type_get, nullptr, "type");
             dukglue_register_property(ctx, &ScRide::name_get, &ScRide::name_set, "name");
```

Comment:
... it's never registered down here. Adding a quick
```
dukglue_register_property(ctx,` &ScRide::classification_get, nullptr, "classification");
```
will fix it up. If you want these to be in the same order as the function declarations above, then put it between `type` and `name`.

Reply:
Thanks, have fixed it!